schoolopy/main.py

```diff
@@ -317,7 +312,8 @@ def get_section(self, section_id):
         :param section_id: ID of section on which to get data.
         :return: Section object.
         """
-        return Section(self._get('sections/%s' % section_id))
+        for section in self.get_sections():
```

Comment:
Are you sure this is the best way to do this?
Also, perhaps could we support passing in a list of section IDs and returning all of them at once, in case for instance people want to get a large number of sections but don't want to make the same `get_sections` query several times?

Reply:
oh sorry i assumed that because the `/sections` endpoint was broken the `/section/{section_id}` would also be broken